core/math/transform_2d.h

```diff
@@ -78,8 +78,8 @@ struct Transform2D {
 
 	void scale(const Size2 &p_scale);
 	void scale_basis(const Size2 &p_scale);
-	void translate(const real_t p_tx, const real_t p_ty);
-	void translate(const Vector2 &p_translation);
+	void translate_local(const real_t p_tx, const real_t p_ty);
```

Comment:
Even though I think the rename can be ok in this context, I believe would also keep translate (and specially translated) as a global version (just offset the origin). There is merit to to having this version too when you want to move the positioning of an object but not its rotation.

In other words, all the code in this PR being ok, but also add back translate/translated() as global.

Reply:
> also add back translate/translated() as global.

`translated` in a global version is already there, right? I've only grouped them below to make them better recognizable as a group. See also unit tests to show that all 6 exist.

In can surely add an in-place `translate` (= global) variant as well. The main reason I didn't is that it is C++ only anyway and not exposed to GDScript.